app/src/main/java/de/tum/in/tumcampusapp/fragments/PdfViewFragment.java

```diff
@@ -37,7 +40,32 @@ public void onResume() {
         pdfView.fromFile(pdf).load();
     }
 
-    public void setPdf(File pdf){
-        this.pdf = pdf;
+    public boolean setPdf(File pdf){
+        if (isFilePdf(pdf)){
+            this.pdf = pdf;
+            return true;
+        }
+        return false;
+    }
+
+    private boolean isFilePdf(File file){
```

Comment:
Please document what this function is doing. Why is it in this class? Why not in an auxiliary helper?